src/hooks/useReusableFaultEvents.tsx

```diff
@@ -18,16 +18,18 @@ export const useReusableFaultEvents = () => {
 
 interface Props extends ChildrenProps {
   treeUri?: string;
+  systemUri?: string;
 }
 
-export const FaultEventsReuseProvider = ({ children, treeUri }: Props) => {
+export const FaultEventsReuseProvider = ({ children, treeUri, systemUri }: Props) => {
```

Comment:
@kostobog  does it makes sense to rename it?

Reply:
@blcham renamed to `ReusableFaultEventsProvider`